packages/data-context/src/DataContext.ts

```diff
@@ -441,10 +420,12 @@ export class DataContext {
       this.actions.dev.watchForRelaunch()
     }
 
+    // We want to fetch the user immediately, but we don't need to block the UI on this
+    this.actions.auth.getUser().catch(() => {
+      // This error is handled internally by getUser, and isn't actionable here
```

Comment:
an empty `catch` block makes this code smelly. either this error should *really* be handled internally by getUser, and not throw, or we should at least log that there is an error being thrown here. otherwise we could be swallowing real errors form getUser.

Reply:
Yeah good point. It doesn't throw, it's just that b/c it's a promise the linting yells b/c it's an unhandled promise. I'll add a `ctx.log(err)` or something here for good measure